app/models/spree/product.rb

```diff
@@ -86,12 +86,12 @@ class Product < ActiveRecord::Base
     validates :name, presence: true
     validates :permalink, presence: true
     validates :price, presence: true, if: proc { Spree::Config[:require_master_price] }
-    validates :shipping_category, presence: true
+    validates :shipping_category_id, presence: true
 
     validates :supplier, presence: true
     validates :primary_taxon, presence: true
-    validates :tax_category, presence: true,
-                             if: proc { Spree::Config[:products_require_tax_category] }
+    validates :tax_category_id, presence: true,
+                                if: proc { Spree::Config[:products_require_tax_category] }
```

Comment:
this wont work in rails 5.2 see here: https://github.com/openfoodfoundation/openfoodnetwork/commit/26ed44412f315470c2b0f2a4feeeafc1097deb9c

Reply:
this change wasn't made by me as it came from origin main pull